### ISSUE TITLE
Failed to publish note to Obsius

### ISSUE THREAD

**Issue by rehanasghar9100** [jonstodle/obsius-obsidian-plugin#14]
![image](https://github.com/jonstodle/obsius-obsidian-plugin/assets/111451359/634a5f36-96df-42ee-9efe-757a72c251b0)

**Comment by rehanasghar9100** [comment]
I have tried to publish simplest note on obsius from each way, but still I'm getting the error: failed to publish note to obsius
I tried to deactivate the other unnecessary plugins, restart the obsidian, creating completely new note but nothing works.

**Comment by tech-wing** [comment]
I have this problem too. Any news?

Thanks for creating this tool, BTW.

**Comment by one2won247** [comment]
I have the same problem too.
Plug-in looks awesome, thank you

**Comment by grnsmn** [comment]
I have the same problem, waiting for solution

**Comment by p11grim** [comment]
same question, waiting

**Comment by Geoffery10** [comment]
I get the message failed to publish but mine seems to publish anyways.

**Comment by grnsmn** [comment]
> I get the message failed to publish but mine seems to publish anyways.

Hello!  I'm curious to know how you were able to ascertain that the note was published despite the 'Failed to publish' message. Did you notice the publication on a specific page, or did you check through a URL or another source?

**Comment by Geoffery10** [comment]
When I try to publish or update I get a failed to publish error message but if I copy the URL and open it in the browser it shows live and fully up to date even though it says it failed.

**Comment by Geoffery10** [comment]
> > I get the message failed to publish but mine seems to publish anyways.
> 
> Hello!  I'm curious to know how you were able to ascertain that the note was published despite the 'Failed to publish' message. Did you notice the publication on a specific page, or did you check through a URL or another source?

When I try to publish or update I get a failed to publish error message but if I copy the URL and open it in the browser it shows live and fully up to date even though it says it failed.

**Comment by elphin** [comment]
Hi Geoffery, where do you copy the url from? I do not see any url after the error...

**Comment by Geoffery10** [comment]
> Hi Geoffery, where do you copy the url from? I do not see any url after the error...

Maybe my issue is slightly different but I have the option to right click the file and copy Obsius url after uploading/updating the file even when it says it failed to upload.

**Comment by CharlysFarias03** [comment]
I'm facing the same error and I ain't seeing the obsius url to copy

**Comment by tech-wing** [comment]
> I have this problem too. Any news?
> 
> Thanks for creating this tool, BTW.

I started using QuickShare which works nicely.

**Comment by josevzs** [comment]
Same here...

**Comment by LauraLangdon** [comment]
Same!

**Comment by gayoungmoney** [comment]
I got the same :(

**Comment by dareiff** [comment]
If you enable developer tools, you can view the network calls and copy the `id` that is returned in response.

**Comment by Nemcade** [comment]
I've been unable to publish anything for a while now and all I can see people having this same problem. And there's no sign of activity other than that. Is this project dead?

**Comment by mcudrio** [comment]
Hi Today, I would publish a note and I have the same message and in the developer's tool I have this message :

TypeError: Cannot read properties of null (reading 'id')
    at Object.eval (plugin:obsius-publish:138:24)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:obsius-publish:47:24)
eval @ VM290 plugin:obsius-publish:144

**Comment by SunnyAureliusRichard** [comment]
Give this a try: https://github.com/alangrainger/share-note

**Comment by Python-Zhou** [comment]
> > I have this problem too. Any news?
> > Thanks for creating this tool, BTW.
> 
> I started using QuickShare which works nicely.

QuickShare is good!

**Comment by jonstodle** [comment]
Hello everyone! I'm sorry I've not responded. Long story short, I had some stuff I needed to work out.

Anyway! I've just published an update to the plugin, and it should now work as before!

Head to Obsidian settings > Community plugins > Current plugins > Update all. That should present an option to update to version `1.3.0` which has the fix.

Thank you for using Obsius and thank you for reporting the issue.
